src/api/routes/tx.ts

```diff
@@ -89,6 +101,31 @@ export function createTxRouter(db: DataStore): RouterWithAsync {
     res.json(response);
   });
 
+  router.getAsync('/tx_list_details', async (req, res, next) => {
+    const txList: string[] = getTxListFromQuery(req.query['tx_list'] as string | undefined);
```

Comment:
I think `getTxListFromQuery` should support the common serialization used for array query params. For example, Express supports automatic parsing of the array formats (also used by web browsers for forms like `<form method="get">`):
```
/?foo=1&foo=2
/?foo[]=1&foo[]=2
```


And I think by default OpenAPI uses the first one `/?foo=1&foo=2`: https://swagger.io/docs/specification/serialization/#query

The function should still support parsing comma-separated strings because that's also a common format.

Reply:
Since `tx_id`s are very long, is there any risk users get a `414 Request-URI Too Long` error when calling this endpoint?

Reply:
@zone117x I have implemented your first approach that makes the endpoint as 
```
/extended/v1/tx/tx_list_details?tx_id=a&tx_id=b
```
where a and b are different `tx_id`s.